velox/expression/tests/ExpressionRunnerTest.cpp

```diff
@@ -83,8 +91,10 @@ DEFINE_string(
     "simplified: evaluate the expression using simplified path and print out "
     "results.\n"
     "query: evaluate SQL query specified in --sql or --sql_path and print out "
-    "results. If --input_path is specified, the query may reference it as "
-    "table 't'.");
+    "results. If --input_paths is specified, the query may reference it as "
+    "table 't'. Note: if selectivity vectors are specified for those inputs "
+    "using --input_selectivity_vector_paths, then those will not be used for "
+    "the query.");
```

Comment:
nit: Maybe simply say "Note: --input_selectivity_vector_paths is ignored in this mode".